src/Time.cpp

```diff
@@ -279,7 +303,22 @@ Time Time::fromString(const std::string& stringTime, Time::Resolution resolution
     tm.tm_isdst = -1;
     time_t time = mktime(&tm);
 
-    return Time(static_cast<int64_t>(time)*UsecPerSec + static_cast<int64_t>(usecs));
+    long tz_offset = 0;
```

Comment:
As far as I can see (just looked at the code), what follows forgets one very important problem: `mktime()` assumes `tm` is in the _current_ time zone and converts it in what it believes that is the _current_ UTC.

I don't think we should allow for anything else than UTC to be stored in the base::Time

A.k.a. the conversion if you'd like to go to UTC is:
~~~
1. strptime: time in "UTC - stringTZOffset"
2. mktime: "UTC - stringTZOffset + localTZOffset"
3. need to add stringTZOffset and _remove_ local offset
~~~